fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -1149,6 +1149,9 @@ class SWIFT_CXX_IMMORTAL_SINGLETON_TYPE ServerKnobs : public KnobsImpl<ServerKno
 	int FETCH_SHARD_BUFFER_BYTE_LIMIT;
 	int FETCH_SHARD_UPDATES_BYTE_LIMIT;
 
+	// Storage Server with Physical Shard
+	bool SS_GET_DATE_MOVE_ID;
```

Comment:
Is SS_GET_DATE_MOVE_ID right? Whats the 'DATE' in SS_GET_DATE_MOVE_ID? Thanks.

Reply:
typo:)